src/sentry/static/sentry/app/views/releases/detail/overview/chart/utils.tsx

```diff
@@ -107,3 +115,338 @@ export function getReleaseEventView(
       });
   }
 }
+
+export function initSessionsBreakdownChartData(): ChartData {
+  return {
+    healthy: {
+      seriesName: sessionTerm.healthy,
+      data: [],
+      color: CHART_PALETTE[3][3],
```

Comment:
Having a local variable for `CHART_PALETTE[3]` might make this easier to maintain down the road.

Reply:
Agreed!